### PR TITLE
maintenance(www): use gatsby-plugin-react-svg for icons

### PR DESCRIPTION
**Pull request by tesseralis**
## Description

* Use `gatsby-plugin-react-svg` to inline SVG icons instead of using `dangerouslySetInnerHTML` and `raw-loader`.
* Standardize how icons are imported (through an index file in the `assets/icons/` directory)
* Prefer passing in the rendered elements as props instead of components
* Convert `featured-sites-icon.js` to `featured-sites.svg`

## Motivation

* less dependence on `dangerouslySetInnerHTML`
* cleaner code

### CODE REVIEW

File: www/src/assets/icons/featured-sites.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="1em" height="1em" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" focusable="false">
```
Comment (fk) [inline www/src/assets/icons/featured-sites.svg:1]: 💯 🙏

File: www/src/components/shared/pullquote.js [hunk 2]
```diff
@@ -168,13 +168,19 @@ const Pullquote = ({ citation, narrow = false, children }) => {
...
-        dangerouslySetInnerHTML={{ __html: QuotationMarkOrnament }}
-      />
+      <QuotationMark>
+        <QuotationMarkOrnament />
+      </QuotationMark>
       <div>
-        <Star dangerouslySetInnerHTML={{ __html: StarOrnament }} />
-        <Star dangerouslySetInnerHTML={{ __html: StarOrnament }} />
-        <Star dangerouslySetInnerHTML={{ __html: StarOrnament }} />
+        <Star>
```
Comment (tesseralis) [inline www/src/components/shared/pullquote.js:175]: @abhishekjakhar If you're still looking for things to refactor, I think this is a good candidate. We can simplify this and other components by converting these styled components into normal components with the icons embedded in them.

Comment (abhishekjakhar) [reply www/src/components/shared/pullquote.js:175]: @tesseralis this sounds good to me, I will start working on it. Thank you.

File: www/gatsby-config.js [hunk 3]
```diff
@@ -258,6 +258,14 @@ module.exports = {
     `gatsby-plugin-twitter`,
     `gatsby-plugin-react-helmet`,
     `gatsby-plugin-sitemap`,
+    {
+      resolve: `gatsby-plugin-react-svg`,
+      options: {
+        rule: {
+          include: /assets\/(guidelines|icons|ornaments)\/.*.svg$/,
```
Comment (muescha) [inline www/gatsby-config.js:265]: was this the intention?
```suggestion
          include: /assets\/(guidelines|icons|ornaments)\/.*\.svg$/,
```

Comment (muescha) [reply www/gatsby-config.js:265]: fixed in  #25765

**Pull request merged**